Libraries/DirectMLX.h

```diff
@@ -3470,9 +3475,43 @@ namespace dml
         return out;
     }
 
-    // 
-    // TODO: NonZeroCoordinates
-    // 
+    struct NonZeroCoordinatesOutputs
+    {
+        Expression count;
+        Expression coordinates;
+    };
+    inline NonZeroCoordinatesOutputs NonZeroCoordinates(
```

Comment:
[nit consistency] Functions that easily fit on a single line are not wrapped.
`constexpr std::size_t size(const T(&array)[N]) noexcept`
`Expression ElementWiseUnary(Expression input, const Optional<DML_SCALE_BIAS>& scaleBias)`